docs/sphinx/features/allocator_accessibility.rst

```diff
@@ -0,0 +1,55 @@
+.. _allocator_accessibility:
+
+=========
+Allocator Accessibility
+=========
+
+The Umpire library provides a variety of :class:`umpire::resource::MemoryResource` s 
+which can be used to create :class:`umpire::Allocator` s depending on what's available on
+your system. The resources are explained more on the `Resources <https://umpire.readthedocs.io/en/develop/tutorial/resources.html>`_
+page.
+
+Additionally, the `platforms <https://github.com/LLNL/Umpire/blob/develop/src/umpire/util/Platform.hpp>`_ that Umpire supports is defined by the CAMP library.
```

Comment:
I think it would be nice to have a link to the relavent CAMP file as well:
```suggestion
Additionally, the `platforms <https://github.com/LLNL/Umpire/blob/develop/src/umpire/util/Platform.hpp>`_ that Umpire supports is defined by the `CAMP <https://github.com/LLNL/camp/blob/master/include/camp/resource/platform.hpp>`_ library.
```

Reply:
Good idea!